content/en/real_user_monitoring/mobile_and_tv_monitoring/setup/ios.md

```diff
@@ -410,24 +413,33 @@ RUM.enable(
 
 ## Track iOS errors
 
-[iOS Crash Reporting and Error Tracking][13] displays any issues in your application and the latest available errors. You can view error details and attributes including JSON in the [RUM Explorer][10].
+[iOS Crash Reporting and Error Tracking][17] displays any issues in your application and the latest available errors. You can view error details and attributes including JSON in the [RUM Explorer][18].
+
+## Supported versions
+
+See [Supported versions][19] for a list operating system versions and platforms that are compatible with the RUM iOS SDK.
```

Comment:
This is the wrong link, it redirects to https://docs.datadoghq.com/logs/log_collection/ios/?tab=cocoapods